frontend/src/components/Review/ReviewTagList/ReviewTagList.tsx

```diff
@@ -40,26 +35,38 @@ const ReviewTagList = () => {
         태그를 골라주세요. (3개)
         <RequiredMark>*</RequiredMark>
       </Heading>
+      <Spacing size={25} />
       <TagListWrapper>
-        {displayedTagList.map(({ id, content }) => {
-          const isSelected = selectedTags.includes(id);
-          const isDisabled = selectedTags.length >= MAX_SELECTED_TAGS && !isSelected;
+        {rev.map((tagGroup) => {
```

Comment:
```suggestion
        {rev.map(({tagType, tags}) => {
```
제안) 이렇게 destructing 해서 쓰는거 어떠신가요?! 저는 이게 더 가독성이 좋은 것 같아요

Reply:
좋아요! 말씀하신 부분 모두 destructing 방식으로 수정했습니다~